scripts/aws-setup.py

```diff
@@ -1,60 +1,11 @@
-#/usr/bin/env python3
+#!/usr/bin/env python3
```

Comment:
The wget path in /docs/Initial-Setup/Configuring-Required-Infrastructure-in-Your-AWS-Account.rst

needs to be updated to point to the new location.

Reply:
Good catch. I forgot about that expecting to be able to grab the single script to download and run.  Rather than changing the path, should we just check in a symlink so that the wget still works with the old path?  Don't know if anyone has that scripted into anything else.

Reply:
Does wget follow symlinks? If it does, I like that solution.

Reply:
GH doesn't serve symlinks as symlinks in the raw.githubusercontent.com URL, it serves them as txt files that have the target of the symlink.  Not what we want.  It looks like those docs may need to be updated to say python3 because I have a feeling that `python` on the FPGA dev AMI may still be python2...  I'm going to double check before updating the docs.

Reply:
Looks like @abejgonzalez already changed the docs on dev to use `./aws-setup.py` to fix the python 2 vs 3 issue.  However, the script had a malformed shebang line and my PR is fixing that.  I'll also update the docs to grab it from the new location with the full-flavor script.